crates/samples/direct2d/src/main.rs

```diff
@@ -156,13 +156,13 @@ impl Window {
             target.SetTarget(clock);
             target.Clear(std::ptr::null());
             self.draw_clock()?;
-            target.SetTarget(previous);
+            target.SetTarget(previous.as_ref());
```

Comment:
Seems to have lost my previous comments on the other commit. Ideally this would be `&previous` if we can make `Borrowed` creatable from `&Option<T>`.

Reply:
Unfortunately, this is not possible as Rust's type system doesn't know that `&Option<T>` does not implement `Into<&'a T>`  and thus we get a conflicting impl error if I add an implementation to make `&Option<T>` work. It's unfortunate, but workable IMO.